examples/triton/setup.sh

```diff
@@ -97,7 +97,26 @@ check() {
         exit 1
     fi
 
-    echo CONSUL=consul.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.joyent.com > _env
+
+    echo '# Prometheus discovery for Triton' > _env
+    echo TRITON_ACCOUNT=${TRITON_ACCOUNT} >> _env
+    echo TRITON_DC=${TRITON_DC} >> _env
+    echo >> _env
+
+    echo '# Prometheus authentication for Triton' >> _env
+    TRITON_CREDS_PATH=/root/.triton
+    echo TRITON_CREDS_PATH=${TRITON_CREDS_PATH} >> _env
+    echo TRITON_CA=$(cat "${DOCKER_CERT_PATH}"/ca.pem | tr '\n' '#') >> _env
+    echo TRITON_CA_PATH=${TRITON_CREDS_PATH}/ca.pem >> _env
+    echo TRITON_KEY=$(cat "${DOCKER_CERT_PATH}"/key.pem | tr '\n' '#') >> _env
+    echo TRITON_KEY_PATH=${TRITON_CREDS_PATH}/key.pem >> _env
+    echo TRITON_CERT=$(cat "${DOCKER_CERT_PATH}"/cert.pem | tr '\n' '#') >> _env
+    echo TRITON_CERT_PATH=${TRITON_CREDS_PATH}/cert.pem >> _env
+    echo >> _env
+
+    echo '# Consul discovery via Triton CNS' >> _env
+    echo CONSUL=prometheus-consul.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.joyent.com >> _env
```

Comment:
This could use the new `triton-docker` helpers for the Consul address. See https://github.com/joyent/triton-docker-cli#notes-and-cautions-about-cns-env-vars and an example in https://github.com/autopilotpattern/nginx/blob/master/examples/triton/docker-compose.yml#L12